src/main/java/com/softserve/kickscootertrip/model/dto/TripDto.java

```diff
@@ -0,0 +1,21 @@
+package com.softserve.kickscootertrip.model.dto;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+public class TripDto {
+    private UUID tripId;
+    private UUID userId;
+    private UUID scooterId;
```

Comment:
Just to confirm: during one trip use can use only one scooter, right? If they take a new one, it'll be a new trip

Reply:
Yes, user need to finish his trip on one scooter to get another one